docs/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -eux -o pipefail
+
+apt-get update
+apt-get install -y curl
+curl -L -o drawio.deb "https://github.com/jgraph/drawio-desktop/releases/download/v${drawio_version}/drawio-${TARGETARCH}-${drawio_version}.deb"
```

Comment:
We ought to reintroduce checksum verification here.

Reply:
I can't think of a way to do this that doesn't involve a lot of obfuscation, so I'm parking it for now.